readthedocs/templates/projects/project_import.html

```diff
@@ -209,6 +209,26 @@ <h1>{% trans "Import a Repository" %}</h1>
         </div>
       {% endblock %}
 
+      {# accounts is a dictionary with the provider as key and a list of connected accounts of that provider #}
+      {% get_social_accounts user as accounts %}
+      {% if accounts %}
+        <h3>{% trans "Filter by Own Repositories" %}</h3>
+        {% for type, list in accounts.items %}
+          {% for account in list %}
+            <li
+              class="remote-org"
+              data-bind="click: function () { $root.set_filter_own('{{ account.provider }}'); }">
+              <img src="{{ account.get_avatar_url }}" class="remote-org-avatar" />
+
+              <!-- TODO: get the proper username for the connected account -->
+              <span>{{ account.user.username }} ({{ account.get_provider_display }})</span>
```

Comment:
I wasn't able to find the username not in our database, but the social account one. How I can get it?

Reply:
Yes, the `extra_data` has this information but I didn't find a generic way to access it (which I suppose that there should be) since bitbucket is `username` and github is `login` and I don't know which one is for gitlab.

I think it's kind of weird that this information is not being accessible in a generic way. `get_avatar_url` exists and does the trick for the avatar; that's why I thought this should exist.